shared/teams/main/index.js

```diff
@@ -135,6 +136,9 @@ class Teams extends React.PureComponent<Props> {
           onCreateTeam={this.props.onCreateTeam}
           onJoinTeam={this.props.onJoinTeam}
         />
+        {this.state.sawChatBanner || this.props.sawChatBanner ? null : (
```

Comment:
i think this should just use this.state. we can add a componentDidUpdate and if the prop value changed it can update the state